### ISSUE TITLE
Uncaught "Connection timeout" error

### ISSUE THREAD

**Issue by vpulim** [ethereumjs/ethereumjs-client#57]
#### Description
The following error is thrown intermittently:
```
UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 5): Error: Connection timeout
```

#### Possible Implementation
I believe this is due to an uncaught error generated by ethereumjs-devp2p when it discovers a peer that doesn't advertise a tcp listen port, but then tries to connect to that peer on that port.

**Comment by vpulim** [comment]
This can closed once this PR is merged and released: https://github.com/ethereumjs/ethereumjs-client/issues/57

**Comment by vpulim** [comment]
That would be great! Thanks

**Comment by holgerd77** [comment]
Ok, just published ``ethereumjs-devp2p`` [v2.5.1](https://github.com/ethereumjs/ethereumjs-devp2p/releases/tag/v2.5.1).

**Comment by vpulim** [comment]
Awesome! Thank you
